sql/telemetry_derived/main_summary_v4/part2.sql

```diff
@@ -1,17 +1,3 @@
-CREATE TEMP FUNCTION udf_json_extract_int_map(input STRING) AS (
-  ARRAY(
-    SELECT
-      STRUCT(
-        SAFE_CAST(SPLIT(entry, ':')[OFFSET(0)] AS INT64) AS key,
-        SAFE_CAST(SPLIT(entry, ':')[OFFSET(1)] AS INT64) AS value
-      )
-    FROM
-      UNNEST(SPLIT(REPLACE(TRIM(input, '{}'), '"', ''), ',')) AS entry
-    WHERE
-      LENGTH(entry) > 0
-  )
-);
-
 CREATE TEMP FUNCTION udf_main_summary_scalars(processes ANY TYPE) AS (
```

Comment:
What is the reason for skipping `udf/main_summary_scalars.sql` in the `publish_persistent_udf` script? Or can that UDF just get published and then having this temporary function here could get removed?

Reply:
I believe the function was too large to publish